models/sessions/scratch/bigquery/snowplow_web_sessions_this_run.sql

```diff
@@ -54,8 +54,9 @@ with session_firsts as (
         mkt_campaign as mkt_campaign,
         mkt_clickid as mkt_clickid,
         mkt_network as mkt_network,
+        regexp_extract(page_url ,r'utm_source_platform=([^?&#]*)') as mkt_source_platform,
```

Comment:
Marginal gains if we use the shorter page_urlquery?
```suggestion
        regexp_extract(page_urlquery ,r'utm_source_platform=([^?&#]*)') as mkt_source_platform,
```